src/cli/Cli.ts

```diff
@@ -35,16 +37,21 @@ export class Cli {
   public currentCommandName: string | undefined;
   private optionsFromArgs: { options: minimist.ParsedArgs } | undefined;
   private commandsFolder: string = '';
+  private showHelpOnFailure: boolean = false;
   private static instance: Cli;
+  private static conf: any;
```

Comment:
Since we know the exact shape of the setting we store, let's define them and not use `any`. This will also allow us to benefit from TypeScript's design-time type safety features and avoid runtime errors.